components/badge.css

```diff
@@ -0,0 +1,105 @@
+.badge-container{
+    position: relative;
+    width: fit-content;
+    height: fit-content;
+    --badge-color-green : #005f00;
+    --badge-color-red : #d10000;
+    --badge-color-yellow : #df7e00;
+}
+
+.status-badge{
+    position: absolute;
+    min-height: 1rem;
+    min-width: 1rem;
+    border-radius: 50%;
+    background-color: green;
```

Comment:
just a sugesstion you would have placed all color variables in root

Also add comments, to separate status and badge stylings

Reply:
Actually I wanted to keep component specific variables in the component, but cool will look into your suggestion.